telegram/_chatinvitelink.py

```diff
@@ -83,6 +84,7 @@ class ChatInviteLink(TelegramObject):
             :tg-const:`telegram.constants.ChatInviteLinkLimit.MIN_MEMBER_LIMIT`-
             :tg-const:`telegram.constants.ChatInviteLinkLimit.MAX_MEMBER_LIMIT`.
         name (:obj:`str`): Optional. Invite link name.
+            0-:tg-const:`telegram.constants.ChatInviteLinkLimit.NAME_LENGTH` characters.
```

Comment:
Telegram API contains no explicit limit for `name` arg/attr

Reply:
`createChatInviteLink` does though. IMO it's okay to mention it here as well